backend/internal/application/service.go

```diff
@@ -740,19 +748,19 @@ func (as *ApplicationService) createApplicationCertificate(cert *certmodel.Certi
 
 	var returnCert *model.ApplicationCertificate
 	if cert != nil {
-		_, svcErr := as.CertService.CreateCertificate(cert)
+		_, svcErr := as.certService.CreateCertificate(cert)
 		if svcErr != nil {
 			if svcErr.Type == serviceerror.ClientErrorType {
 				retErr := serviceerror.ServiceError{
```

Comment:
Shall we refactor these usages to use `CustomServiceError()` from serviceerror package [1]?
Check other places as well. 

[1] https://github.com/asgardeo/thunder/blob/23fd28ef6eb744ac841a0fe7e48531450517cd88/backend/internal/system/error/serviceerror/error.go#L41

Reply:
Fixed.